src/app/movie-list/movie-list.component.html

```diff
@@ -1,4 +1,5 @@
-<app-movie-search (searchValue)="getMovies($event)"></app-movie-search>
+<app-movie-search (searchValue)="setSearchValue($event)"></app-movie-search>
+<app-pagination class="center-align" (pageChange)="setPage($event)" [totalPages]="totalPages" [page]="page"></app-pagination>
```

Comment:
Add *ngIf="page && totalPages" here. We should not show paginator without pagination data

Reply:
Done